src/renderer/components/+user-management/+role-bindings/__tests__/dialog.test.tsx

```diff
@@ -39,6 +43,10 @@ describe("RoleBindingDialog tests", () => {
         uid: "1",
       },
     })];
+
+    const di = getDiForUnitTesting();
+
+    render = renderFor(di);
```

Comment:
Is it related to PR's somehow?

Reply:
Only related because tests will fail without this (because NamespaceSelect now uses DI).